model/src/main/kotlin/licenses/PackageLicenseInfoProvider.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.licenses
+
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.Provenance
+import org.ossreviewtoolkit.utils.ProcessedDeclaredLicense
+
+/**
+ * A provider for [PackageLicenseInfo]s created from an [ortResult].
+ */
+class PackageLicenseInfoProvider(val ortResult: OrtResult) {
```

Comment:
I'd omit "Package" from the class name as otherwise it sounds as it there would also be a "ProjectLicenseInfoProvider".

Reply:
Done.